mod/wikirate/set/right/tab_content.rb

```diff
@@ -0,0 +1,19 @@
+format :html do
+  view :core do |_args|
+    # show the content based on the url parameter
+    # tabs: metric, topic, company, note, reference, overview
+    tab = Env.params['tab']
+    left_name = card.cardname.left
+    card_tab_name =
+      if !tab.nil?
+        "#{left_name}+#{tab}_page"
+      else
+        "#{left_name}+metric_page"
+      end
+    if (content_card = Card.fetch card_tab_name)
+      subformat(content_card).render_content
+    else
+      ''
```

Comment:
I might put an error like "no such tab: #{tab}".  This should only happen if we have a bad tab name, right?